cmd/maya-apiserver/app/server/volume_endpoint_v1alpha1.go

```diff
@@ -35,20 +33,20 @@ func (s *HTTPServer) volumeV1alpha1SpecificRequest(resp http.ResponseWriter, req
 	case "PUT", "POST":
 		return volOp.create()
 	case "GET":
-		return volOp.get()
+		return volOp.httpGet()
+	case "DELETE":
+		return volOp.httpDelete()
 	default:
 		return nil, CodedError(405, ErrInvalidMethod)
 	}
 }
 
-// TODO Move the delete out to HTTP DELETE
-//
 // get deals with HTTP GET request
-func (v *volumeAPIOpsV1alpha1) get() (interface{}, error) {
+func (v *volumeAPIOpsV1alpha1) httpGet() (interface{}, error) {
```

Comment:
Can we return specific GO struct schema instead of interface ?

Reply:
will do for some of the methods in this file.